schema/base-model.js

```diff
@@ -82,6 +82,14 @@ class BaseModel extends Model {
     return query
       .orderBy(sort.column, sort.ascending === 'true' ? 'asc' : 'desc');
   }
+
+  static validate(data) {
+    try {
+      this.fromJson(data);
+    } catch (error) {
+      return error;
```

Comment:
Should we convert to a schema-namespaced `ValidationError` here?

Reply:
I'm gunna say, yeah, that'd be nice :)